package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@ginger.io/jay-z",
-  "version": "0.0.13",
+  "version": "0.0.15",
```

Comment:
The version is being bumped twice because Vault is using JayZ `v0.0.14`, and we want to increment the version again to account for the nonce fix.